### ISSUE TITLE
Support QR code entry

### ISSUE THREAD

**Issue by mcandre** [JumpMaster/QuickAuth#24]
Many 2FA-enabled websites do not allow the private token to be displayed or redisplayed, such as when authenticating Pebble after authenticating another device. However, the QR codes are generally allowed to be displayed and redisplayed for authenticating multiple devices.

So it would be helpful if QuickAuth supported tokens added via QR code images, using the phone's camera to capture them.

I'm currently mitigating this by deleting my old tokens/QR codes, and setting up 2FA from scratch on each account, making sure to copy the manual token into QuickAuth before it is made unavailable as part of the 2FA setup process on different websites.

**Comment by JumpMaster** [comment]
I completely agree and have even made a version of the html configuration page that does exactly that. I unfortunately the Pebble app does not have camera permissions and they have ignored my requests to add it. Secondly Safari for ios doesn't support access to the camera. IE does but not Safari.

Please email Pebble asking for camera support within the configuration paves. If more people ask they might listen.

Kev

**Comment by mcandre** [comment]
Posted.

https://forums.getpebble.com/discussion/30116/please-allow-html-configuration-pages-to-access-the-camera?new=1
